<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pommo</class>
 <widget class="QWidget" name="pommo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>613</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>24</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(:/img/8.jpg);</string>
  </property>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	font:bold;
	font-size:30px;
	
	border-style:outset;
	border-radius:5px;
	border-color:black;
	background-color:rgb(65, 129, 172);
	color:white;
}
QPushButton:hover{
	color:rgb(22, 31, 70);
	background-color:white;
border-color:white;
}</string>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	font:bold;
	font-size:30px;
	
	border-style:outset;
	border-radius:5px;
	border-color:black;
	background-color:rgb(65, 129, 172);
	color:white;
}
QPushButton:hover{
	color:rgb(22, 31, 70);
	background-color:white;
}</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>341</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font-family:Monaco;

border-width:12px;

border-color:rgb(40, 80, 106);


</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>260</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	font:bold;
	font-size:30px;
	
	border-style:outset;
	border-radius:5px;
	border-color:black;
	background-color:rgb(65, 129, 172);
	color:white;
}
QPushButton:hover{
	color:rgb(22, 31, 70);
	background-color:white;
border-color:white;
}</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>871</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>610</y>
     <width>871</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>141</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSpinBox" name="sessionTimeSpinBox">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="TotalSessionsSpinBox">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="breakTimeSpinBox">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	
	font:bold;
	font-size:30px;
	
	border-style:outset;
	border-radius:5px;
	border-color:black;
	background-color:rgb(65, 129, 172);
	color:white;
}
QLabel:hover{
	color:rgb(255, 255, 255);
	background-color:rgb(60, 120, 158);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>Session Time:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	
	font:bold;
	font-size:30px;
	
	border-style:outset;
	border-radius:5px;
	border-color:black;
	background-color:rgb(65, 129, 172);
	color:white;
}
QLabel:hover{
	color:rgb(255, 255, 255);
	background-color:rgb(36, 71, 95);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Total sessions:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	
	font:bold;
	font-size:30px;
	
	border-style:outset;
	border-radius:5px;
	border-color:black;
	background-color:rgb(65, 129, 172);
	color:white;
}
QLabel:hover{
	color:rgb(255, 255, 255);
	background-color:rgb(36, 71, 95);
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Break Time:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>90</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font-size:40;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
